collectd/map.jinja

```diff
@@ -8,6 +8,7 @@
         'read_interval': 60,
         'file_logging': True,
         'remote_collector': False
+        'use_fqdn': True,
```

Comment:
I wonder if `use_fqdn` should also be added https://github.com/tcpcloud/salt-formula-collectd/blob/master/metadata/service/client/init.yml. Just for the sake of not hiding this parameter.